src/planning/strategies/tests/resolve-recipe-test.ts

```diff
@@ -12,11 +12,19 @@ import {Manifest} from '../../../runtime/manifest.js';
 import {ResolveRecipe} from '../../strategies/resolve-recipe.js';
 
 import {StrategyTestHelper} from '../../testing/strategy-test-helper.js';
+import {TestVolatileMemoryProvider} from '../../../runtime/testing/test-volatile-memory-provider.js';
+import {RamDiskStorageDriverProvider} from '../../../runtime/storageNG/drivers/ramdisk.js';
 import {Entity} from '../../../runtime/entity.js';
 
 const {createTestArc, onlyResult, theResults, noResult} = StrategyTestHelper;
 
 describe('resolve recipe', () => {
+  let memoryProvider;
+  beforeEach(() => {
+    memoryProvider = new TestVolatileMemoryProvider();
```

Comment:
SimpleVolatileMemoryProvider (in volatile.ts) and TestVolatileMemoryProvider are practically identical. 
Just curios, do we need both?

Reply:
I made them separate intentionally because I figure at some point we'll likely have tests that want to scrutinize the volatile memory values, so though TestVolatileMemoryProvider doesn't have any accessors currently, things are set up so it's easy to add them.

We could have the Test one extend the Simple one, or just use Simple and delete the Test one until we need it, but I figured it was fine the way it is since it's very little code and I thought it worth doing the test named version one now to save others the hassle down the road.